sycl/include/CL/sycl/detail/queue_impl.hpp

```diff
@@ -48,7 +48,7 @@ class queue_impl {
              QueueOrder Order, const property_list &PropList)
       : queue_impl(Device,
                    detail::getSyclObjImpl(
-                       context(createSyclObjFromImpl<device>(Device))),
+                       context(createSyclObjFromImpl<device>(Device), {}, true)),
```

Comment:
Nit: `{}` and `true` seem to me some "magic" values. Can you please add comments here?